### ISSUE TITLE
[DOC] - Document a11y auditing process

### ISSUE THREAD

**Issue by trallard** [Quansight-Labs/jupyter-a11y-mgmt#31]
## Background

Produce documentation on our process for auditing interactive computing software.

This should be part of our a11y framework

**Comment by trallard** [comment]
TODO:  cleanup and run other keyboard navigation (make it more robust)
